model/shared/src/main/scala/explore/optics/all.scala

```diff
@@ -176,11 +178,27 @@ object all extends ModelOptics {
   def atMapWithDefault[K, V](k: K, default: => V): Lens[Map[K, V], V] =
     atMap.at(k).andThen(getWithDefault(default))
 
-  implicit def atTreeSeqMap[K, V]: At[TreeSeqMap[K, V], K, Option[V]] =
+  given treeSeqMapAt[K, V]: At[TreeSeqMap[K, V], K, Option[V]] =
     At(i =>
       Lens((_: TreeSeqMap[K, V]).get(i))(optV => map => optV.fold(map - i)(v => map + (i -> v)))
     )
 
-  implicit def indexTreeSeqMap[K, V]: Index[TreeSeqMap[K, V], K, V] =
-    fromAt(atTreeSeqMap)
+  given treeSeqMapIndex[K, V]: Index[TreeSeqMap[K, V], K, V] = fromAt
+
+  // Not used after all, is it worth keeping?
```

Comment:
Probably not, unless you foresee a future usage. On the other hand, I wouldn't find it easy to reproduce...

Reply:
It is actually used, the comment is outdated! :D

Reply:
For the moment we are not using `TreeSeqMap`s directly, only through `KeyedIndexedList`, which has its own `FilterIndex` instance. This was copied and adapted from `monocle`'s implementation for `Map`s, it should be trivial to do it again if need be.